tests/tcp.rs

```diff
@@ -1041,3 +1041,33 @@ fn socket_to_nonexistent_node() -> Result {
     });
     sim.run()
 }
+
+#[test]
+fn high_host_latency() -> Result {
```

Comment:
I'm pretty comfortable with this existing test: `override_link_latency`. Did you see some gaps that are covered by this new one?

Reply:
Ahh I missed that test, no that definitely covers it!